src/provenance_gc.rs

```diff
@@ -1,6 +1,7 @@
 use either::Either;
 use rustc_data_structures::fx::FxHashSet;
 
+use crate::shims::io_error::IoError::*;
```

Comment:
No please don't do this in the entire file, it pollutes the namespace too much. If you want this for a `match`, only import the variants locally inside that function.